plugins/opds.koplugin/opdsbrowser.lua

```diff
@@ -695,7 +695,40 @@ function OPDSBrowser:downloadFile(item, filename, remote_url)
     end
 end
 
-function OPDSBrowser:streamPages(item, remote_url, count)
+-- This function shows a dialog with input fields
+-- for entering information for Page streaming from.
+-- an input page number
+function OPDSBrowser:jumpToPage(viewer)
+    local input_dialog
+    input_dialog = InputDialog:new{
+        title = _("Enter Page Number"),
+        input = "",
+        input_hint = "Page #",
+        buttons = {
+            {
+                {
+                    text = _("Cancel"),
+                    id = "close",
+                    callback = function()
+                        UIManager:close(input_dialog)
+                    end,
+                },
+                {
+                    text = _("Stream"),
+                    is_enter_default = true,
+                    callback = function()
+                        UIManager:close(input_dialog)
+                        viewer:switchToImageNum(tonumber(input_dialog:getInputValue()))
```

Comment:
You can add `input_type == "number"`, `getInputValue` makes `tonumber` for you.

Reply:
> This will also switch the keyboard to the symbols/numbers layer.

I Did not know this, I will change it in the next commit!

Reply:
Leaving a note for myself, this is done.